tracing/src/span.rs

```diff
@@ -720,10 +720,21 @@ impl Span {
     ///
     /// If this span is disabled, or the resulting follows-from relationship
     /// would be invalid, this function will do nothing.
-    pub fn follows_from(&self, from: impl for<'a> Into<Option<&'a Id>>) -> &Self {
+    ///
+    /// # Example
+    ///
+    /// ```
+    /// # use tracing::{span, Id, Level, Span};
+    /// # fn main() {
+    /// let span = span!(Level::INFO, "hello!");
+    /// let curr = Span::current();
+    /// span.follows_from(curr.id());
```

Comment:
It should be possible to just say
```rust
let curr = Span::current();
span.follows_from(curr);
```
and 
```
span.follows_from(&curr);
```
as well — can we make sure this is the case?

Reply:
Sure.